config.json

```diff
@@ -11,46 +11,43 @@
       {
         "name": "bob",
         "node_type": "validator",
-        "prometheus": true
+        "prometheus": false
 
       }
     ]
   },
 
   "parachains": [
-    {
-      "name":"acala",
-      "nodes": [
-        {
-          "name": "alice",
-          "node_type": "validator",
-          "prometheus": false
-
-        },
-        {
-          "name": "bob",
-          "node_type": "full",
-          "prometheus": true
-        }
-      ]
-    },
     {
       "name":"frequency",
       "nodes": [
         {
           "name": "alice",
-          "node_type": "validator",
-          "prometheus": false
+          "node_type": "collator",
+          "prometheus": false,
+          "key":{
+            "private_phrase":"dizzy rose offer wall social glory debris gift govern seminar almost bicycle" ,
+            "public_key":"5G3vA9UC6J7hPN5Jqfiz4qFNqYAfahHxffbuQf6dcZZx3Yx6"
+          }
 
         },
         {
           "name": "bob",
           "node_type": "full",
-          "prometheus": true
+          "prometheus": false,
+          "key": {
+            "private_phrase":"diamond crane pioneer aisle conduct media news cup price olive dust light" ,
+            "public_key":"5GgwsM48CCT7oPUyFTTpE5L1t7Fjpe7D8SDNKxJxug15YEKy"
+          }
         }
-      ]
+      ],
+      "sudo_key": {
```

Comment:
without sudo key throwing error, can you make the sudo key as optional
![Uploading Screenshot 2024-02-13 at 5.53.37 PM.png…]()